package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "hexo-generator-archive",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

```suggestion
  "version": "2.0.0",
```

No need to change the version number